markymark/Classes/Layout Builders/UIView/Block Builders/Inline/InlineAttributedStringViewLayoutBlockBuilder.swift

```diff
@@ -7,10 +7,13 @@ import UIKit
 
 class InlineAttributedStringViewLayoutBlockBuilder: LayoutBlockBuilder<UIView> {
 
-    fileprivate let converter : MarkDownConverter<NSMutableAttributedString>
+    let urlOpener: URLOpener?
```

Comment:
you are injecting this urlOpener through the init, as it's a let so not possible to set from outside.
Can it be private?
```suggestion
    private let urlOpener: URLOpener?
```